implementations/micrometer-registry-elastic/src/main/java/io/micrometer/elastic/ElasticMeterRegistry.java

```diff
@@ -254,9 +257,21 @@ private String indexName() {
 
     // VisibleForTesting
     Optional<String> writeMeter(Meter meter) {
+        Iterable<Measurement> measurements = meter.measure();
+        List<Double> values = new ArrayList<>();
+        for (Measurement measurement : measurements) {
+            double value = measurement.getValue();
+            if (!Double.isFinite(value)) {
+                return Optional.empty();
```

Comment:
Could we just skip the measurement with the non-finite value, rather than all of the measurements?

Reply:
@shakuzen I chose this way due to [this internal restriction](https://github.com/micrometer-metrics/micrometer/blob/master/micrometer-core/src/main/java/io/micrometer/core/instrument/Meter.java#L50-L51) but publishing it sounds like a different concern, so I pushed 7700492 as you suggested.